### DISCUSSION TITLE
Copying Feature Service

### DISCUSSION THREAD

**Discussion by NRW-INSPIRE** [EsriPS/arcgis-assistant-feedback#74] [Q&A]
When I try to copy a feature service, i see this message:
![image](https://github.com/EsriPS/arcgis-assistant-feedback/assets/70281435/43ed3e40-7a1b-4c21-a8be-6448f8067122)
What does it mean?

**Comment by jpeterson** [comment]
Hey @NRW-INSPIRE, feature services are represented by two things: an **item** and a **service** (i.e. the FeatureServer). When copied via Assistant, the item gets copied, but it still references the original service.

Note that we are working on adding the ability to clone feature services (copying both the item and service).

More details here: https://guide.assistant.esri-ps.com/docs/working-with-items#services-feature-service-map-service-image-service-etc
